odo/backends/aws.py

```diff
@@ -70,9 +73,11 @@ def __init__(self, uri, s3=None, aws_access_key_id=None,
             self.s3 = s3
         else:
             self.s3 = get_s3_connection(aws_access_key_id=aws_access_key_id,
-                                        aws_secret_access_key=aws_secret_access_key)
+                                        aws_secret_access_key=aws_secret_access_key,
+                                        **kwargs)
         try:
             bucket = self.s3.get_bucket(self.bucket,
+                                        validate=False,
```

Comment:
This shouldn't be necessary with the `**filter_kwargs(...)` unpacking on the next line -- if the user passes in `validate=False` to the constructor, it should make it through via `filter_kwargs()`.

That said, it may be necessary to do some work to ensure top-level `kwargs` make it all the way down to this function.

I wouldn't want to hardcode `validate=False` from skimming this: http://docs.pythonboto.org/en/latest/s3_tut.html#accessing-a-bucket